frontend/device/generic/device.lua

```diff
@@ -54,6 +54,7 @@ local Device = {
     hasExternalSD = no, -- or other storage volume that cannot be accessed using the File Manager
     canHWDither = no,
     canHWInvert = no,
+    canAnimate = no,
```

Comment:
(Feels a little too generic, no ? I guess we could animate other things :) may be canDoSwipeAnimation ?)